subsys/mgmt/mcumgr/grp/fs_mgmt/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright (c) 2018-2021 mcumgr authors
+# Copyright (c) 2022 Laird Connectivity
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# File System management group public API is exposed by MCUMGR API
+# interface, when File System management is enabled.
+add_library(MCUMGR__grp__fs src/zephyr_fs_mgmt.c src/fs_mgmt.c)
+
+target_sources_ifdef(CONFIG_FS_MGMT_CHECKSUM_HASH
+	MCUMGR__grp__fs PRIVATE src/hash_checksum_mgmt.c
+)
+target_sources_ifdef(CONFIG_FS_MGMT_CHECKSUM_IEEE_CRC32
+	MCUMGR__grp__fs PRIVATE src/hash_checksum_crc32.c
+)
+target_sources_ifdef(CONFIG_FS_MGMT_HASH_SHA256
+	MCUMGR__grp__fs PRIVATE src/hash_checksum_sha256.c)
```

Comment:
Follow usage of other in-tree modules with single underscore and lowercase names - for all cmake files.
```
./tests/bluetooth/host/CMakeLists.txt:add_library(host_mocks STATIC
/samples/tfm_integration/tfm_secure_partition/dummy_partition/CMakeLists.txt:add_library(tfm_app_rot_partition_dp STATIC)
./samples/application_development/external_lib/CMakeLists.txt:add_library(mylib_lib STATIC IMPORTED GLOBAL)
./cmake/linker/armlink/target.cmake:    add_library(armlink_steering OBJECT ${STEERING_C})
./cmake/linker/arcmwdt/target.cmake:  add_library(code_relocation_source_lib  STATIC ${MEM_RELOCATION_CODE})
```

Reply:
Done.